src/main/java/PageObjects/ChangePasswordPage.java

```diff
@@ -7,31 +7,36 @@
 public class ChangePasswordPage {
 
     //Locator
-    private final By loc_txtCurrentPassword = By.xpath("//input[@id=\"currentPassword\"]");
-    private final By loc_txtNewPassword = By.xpath("//input[@id=\"newPassword\"]");
-    private final By loc_txtConfirmPassword = By.xpath("//input[@id=\"confirmPassword\"]");
-    private final By loc_btnChangePassword = By.xpath("//input[@value=\"Change Password\"]");
-    private final By loc_changePasswordSuccessMsg = By.xpath("//p[@class='message success']");
+    private final By txtCurrentPassword = By.xpath("//input[@id='currentPassword']");
```

Comment:
For txtCurrentPassword, txtNewPassword, txtConfirmPassword element, capture By.id directly